bpfman/src/multiprog/xdp.rs

```diff
@@ -157,11 +157,13 @@ impl XdpDispatcher {
                 .unwrap();
         } else {
             let flags = self.mode.as_flags();
-            let link = dispatcher.attach(&iface, flags).unwrap();
+            let link = dispatcher
+                .attach(&iface, flags)
+                .map_err(|e| BpfmanError::Error(format!("dispatcher attach: {e}")))?;
```

Comment:
```suggestion
                .map_err(|e| BpfmanError::Error(format!("dispatcher attach failed on interface {iface}: {e}")))?;
```

Reply:
We might want to actually have a custom `BpfmanError::AttachError` variant here that wraps an aya error, since we could potentially use that everywhere we call attach